### ISSUE TITLE
Remove user's email address from the moderator "new user registration" emails

### ISSUE THREAD

**Issue by dfabulich** [iftechfoundation/ifdb-suggestion-tracker#165]
Right now, the "new user registration" emails to moderators include the user's email address. That's bad, because it means that IFDB users' private information now resides in all moderators' email archives. If we decide to "fire" a moderator, there's no way to claw that information back.

Occasionally, it can be informative to see the user's email address (e.g. gmail addresses containing a lot of dots are typically spammers) but Akismet's spammer identification system can typically identify those spammy emails for us. (I don't think we've ever seen an email with lots of dots that Akismet didn't identify as spam.)

I don't think we need more than the Akismet judgment and the StopForumSpam counts to make decisions on these emails; if we do need them, we can just include a link in the email to click through to the website to see the actual email address.

**Comment by curiousdannii** [comment]
> That's bad, because it means that IFDB users' private information now resides in all moderators' email archives.

That's a fair concern. Ideally there'd be a moderator control panel in the IFDB software itself so this can all be better managed without going offsite.

**Comment by dfabulich** [comment]
There _is_ a moderator control panel, but it's not as convenient as clicking links in the emails.

**Comment by fos1** [comment]
I disagree. Emails are not protected personally identifiable information.
They can be changed at will and faked. It is a necessary identifier. It is
provided by the user and they know at registration that it will be used for
administrative purposes. A moderator cannot adequately screen users.

On my own very small forum, I get approximately 100 severe spammers that
slip through the automated spam systems. A manual check is the only way to
begin to ferret them out.

It could be argued that we have too many moderators. Either the moderators
are going to have authority or they are not.

**Comment by qdacsvx** [comment]
As someone who uses ifdb, I might fear if I'm banned from one IF website, will my email be passed around and I won't be treated fairly on other IF websites?

**Comment by fos1** [comment]
The spam filtering software systems add you to their spam lists. Moderators
are pretty useless if they can't verify your email address. I do it on my
discussion forums everyday. most listed spammers have their IP addresses,
user names, and email addresses listed as much as 1000+ times. About 100
confirmed spammers try to register on each of my small sites each month on
average. That is why they are called spammers. Forum (social site)
administrators have to have a means to identify the miscreants.

**Comment by dfabulich** [comment]
I wanted to clarify that in no case would the emails be altogether hidden from moderators. Since you have to click through to the website to take action on moderator emails anyway, I just meant that you'd have to actually go to the website to see the email address, so it wouldn't just be sitting around in your email archives automatically.

**Comment by dfabulich** [comment]
It's not clear whether we should ever do this, but we certainly shouldn't do it until we do #263.
